### ISSUE TITLE
Being able to scale the object separately from tile image in Tile Objects

### ISSUE THREAD

**Issue by pouwelsjochem** [mapeditor/tiled#3129]
Currently a `Tile Object` always spawns an object with the same size as the referenced tile. We can then scale the object which stretches the tile image (sometimes unwanted because you want Tiled to represent the actual objects in the map, not a resized version).

In a lot of grid based games the object you interact with/walk against is bigger or smaller than the pixels you see on screen. I would love to be able to use Tile Objects with a different 'collision'.

Either in a `Template` or regular `Tile Object` would solve the issue for me, but I think `Templates` are the best fit to define the collision in.

One possible implementation would be adding a boolean to a `Template` which would disable the scaling/repositioning of the tile image. This could then allow freely moving around and scaling the object based on the image. 
Basically the same as the Tile Collision Editor we already have (but with just 1 object)

![Screenshot 2021-09-10 at 09 45 15](https://user-images.githubusercontent.com/19981050/132820159-8b660c44-8c39-4151-a507-12574d396f00.png)
